client/src/product-components/AddToCart/AddToCart.jsx

```diff
@@ -47,39 +57,53 @@ class AddToCart extends React.Component {
     });
   }
 
+  handleAddToCart() {
+    axios.post('/cart', {
+      sku_id: this.props.sizeList[this.state.selectedIndex].sku,
+    })
+    .then(function (response) {
+      console.log(response);
+    })
+    .catch(function (error) {
+      console.log(error);
+    });
+
+  }
+
   render() {
     // console.log(this.state.quantityList, '================this.state.quantityList');
-    // console.log(this.props.sizeList, '================this.props.sizeList');
+    // console.log(this.props.sizeList, this.state.selectedIndex, '================this.props.sizeList');
     //console.log(this.props.sizeList, this.state.selectedIndex, 'in render');
     return (
       <div>
-        <div className="SizeSelector">
+        <div  className="SizeSelector">
           Size:
-          <select value={this.state.sizeValue} onChange={this.handleSizeChange.bind(this)}>
+          <select value={this.state.sizeValue} disabled={this.props.sizeOutOfStock} onChange={this.handleSizeChange.bind(this)}>
+            <option value={'SelectSize'}>{this.props.sizeOutOfStock ? 'OUT OF STOCK' : 'Select Size'}</option>
```

Comment:
"Select Size" should definitely be a variable at this point, 3rd time I've seen it and this usage is slightly different than the first two.